assets/javascripts/discourse/connectors/after-d-editor/ai-helper-context-menu.js

```diff
@@ -76,7 +77,29 @@ export default class AiHelperContextMenu extends Component {
 
     prompts = prompts
       .filter((p) => p.location.includes("composer"))
-      .filter((p) => p.name !== "generate_titles");
+      .filter((p) => p.name !== "generate_titles")
+      .map((p) => {
+        // AI helper by default returns interface locale on translations
+        // Since we want site default trnaslations (and we are using: force_default_locale)
+        // we need to replace the translated_name with the site default locale name
+        const siteLocale = this.siteSettings?.default_locale;
+        const availableLocales = JSON.parse(
+          this.siteSettings.available_locales
+        );
+        const locale = availableLocales.find((l) => l.value === siteLocale);
+        const translatePromptName = I18n.t(
+          "discourse_ai.ai_helper.context_menu.translate_prompt",
+          {
+            language: locale?.name,
+          }
+        );
+        const translatedName = locale ? translatePromptName : "Translate";
+
+        if (p.name === "translate") {
+          return { ...p, translated_name: translatedName };
+        }
```

Comment:
This code block is a bit weird. Is it even possible that `this.siteSettings` can be `null` when this is called? If that's the case, then wouldn't `JSON.parse(this.siteSettings.available_locales);` fail as well? Also, is it really possible that the `siteLocale` can't be found in `availableLocales`?